src/main/java/org/carlmontrobotics/subsystems/Camera.java

```diff
@@ -0,0 +1,29 @@
+package org.carlmontrobotics.subsystems;
+import edu.wpi.first.cscore.CvSink;
+import edu.wpi.first.cscore.CvSource;
+import edu.wpi.first.cscore.MjpegServer;
+import edu.wpi.first.cscore.UsbCamera;
+import edu.wpi.first.cscore.UsbCameraInfo;
+import edu.wpi.first.util.PixelFormat;
+public class Camera {
+    public Camera(){
+        UsbCamera usbCamera = new UsbCamera("USB Camera 0", 0);
+        MjpegServer mjpegServer1 = new MjpegServer("serve_USB Camera 0", 1
+        );
+        mjpegServer1.setSource(usbCamera);
+
+        // Creates the CvSink and connects it to the UsbCamera
+        CvSink cvSink = new CvSink("opencv_USB Camera 0");
+        cvSink.setSource(usbCamera);
+
+        // Creates the CvSource and MjpegServer [2] and connects them
+        CvSource outputStream = new CvSource("Blur", PixelFormat.kMJPEG, 640, 480, 30);
+        MjpegServer mjpegServer2 = new MjpegServer("serve_Blur", 2);
+        mjpegServer2.setSource(outputStream);
+
+
+        //OR
+        
+    }
+    
+}
```

Comment:
Not clear if this is testing cruft that can be removed or not. At the moment, the camera is created by `SensorFactory.configureCamera()`, not by this subsystem.